src/plugins/stepsform/stepsform.js

```diff
@@ -0,0 +1,217 @@
+/**
+ * @title WET-BOEW Form validation
+ * @overview Provide ability for a form to be broken into steps.
+ * @license wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
+ * @author @kodecount
+ */
+( function( $, window, document, wb ) {
+"use strict";
+
+/*
+ * Variable and function definitions.
+ * These are global to the plugin - meaning that they will be initialized once per page,
+ * not once per instance of plugin on the page. So, this is a good place to define
+ * variables that are common to all instances of the plugin on a page.
+ */
+var componentName = "wb-steps",
+	selector = "." + componentName,
+	initEvent = "wb-init" + selector,
+	$document = wb.doc,
+	idCount = 0,
+	i18n, i18nText,
+
+	/**
+	 * @method init
+	 * @param {jQuery Event} event Event that triggered the function call
+	 */
+	init = function( event ) {
+
+		// Start initialization
+		// returns DOM object = proceed with init
+		// returns undefined = do not proceed with init (e.g., already initialized)
+		var eventTarget = wb.init( event, componentName, selector ),
+			elmId;
+
+		if ( eventTarget ) {
+			elmId = eventTarget.id;
+
+			// Ensure there is a unique id on the element
+			if ( !elmId ) {
+				elmId = componentName + "-id-" + idCount;
+				idCount += 1;
+				eventTarget.id = elmId;
+			}
+
+			// Only initialize the i18nText once
+			if ( !i18nText ) {
+				i18n = wb.i18n;
+				i18nText = {
+					prv: i18n( "prv" ),
+					nxt: i18n( "nxt" )
+				};
+			}
+
+			// remove submit button (user should add one for no script purposes)
+			$( eventTarget ).find( "input[type=submit]" ).remove();
+
+			// find all fieldsets, hide required content, and add appropriate buttons
+			var fieldsets = eventTarget.getElementsByTagName( "FIELDSET" ),
+				settings = wb.getData( eventTarget, componentName ),
+				btnValue = ( settings && settings.submitBtnVal ) ? settings.submitBtnVal : "X",
```

Comment:
- [ ] i18n Text (add "Submit" as default button text (enabling tmpl-submit (not sure about this)))